common/interfaces/user_interface/ui_logo.c

```diff
@@ -58,27 +58,19 @@
  */
 #include "ui_logo.h"
 
-void logo_scr_init(const uint16_t delay_in_ms) {
-  lv_obj_t *label = lv_label_create(lv_scr_act(), NULL);
-  lv_label_set_align(label, LV_LABEL_ALIGN_CENTER);
-  lv_obj_align(
-      label,
-      NULL,
-      LV_ALIGN_IN_TOP_LEFT,
-      5,
-      25);    // Alignment may have to be manually changed according to text
-
-  static lv_style_t style;
-  lv_style_copy(&style, lv_theme_get_mono()->style.label.prim);
-  style.text.font = &my_logo;    // This is a file in lvgl/src/lv_font
-  lv_label_set_style(label, LV_LABEL_STYLE_MAIN, &style);
+#include "events.h"
+#include "ui_multi_instruction.h"
 
-  lv_label_set_text(label, MY_LOGO_SYMBOL);
+void logo_scr_init(const uint16_t delay_in_ms) {
+  LV_IMG_DECLARE(cypherock_logo);
```

Comment:
this appears to be a declaration. take out of the function.

Reply:
updated